release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -196,6 +196,17 @@ By default, the following network connections are checked:
 ** the internal API load balancer
 ** the external API load balancer
 
+[id="ocp-4-6-operators"]
+=== Operators
+
+[id="ocp-4-6-metering-operator"]
+==== Support for retention period for metering Reports
+
+You can now set a retention period on a metering Report. The metering Report CustomResource
+has a new `expiration` field. If the `expiration` duration value is set on a Report,
```

Comment:
@lbarbeevargas @bentito perhaps an example here would benefit the end user.

Reply:
I thought about that too but I think just release notes here, right?

Reply:
I can add a link to the docs if the user wants more information and an example.